src/Twig/MediaRuntime.php

```diff
@@ -150,7 +152,10 @@ private function getMedia($media): ?MediaInterface
             return null;
         }
 
-        if (MediaInterface::STATUS_OK !== $media->getProviderStatus()) {
+        if (
+            MediaInterface::STATUS_OK !== $media->getProviderStatus() &&
+            !\in_array(strtolower(pathinfo((string) $media->getName(), \PATHINFO_EXTENSION)), self::EXCLUDED_FORMATS, true)
```

Comment:
Same

Reply:
made the proposed changes, but other errors appeared in the tests